packages/cli/src/commands/open.js

```diff
@@ -6,10 +6,12 @@ import openWebPage from 'open';
 import { E } from '@endo/far';
 
 import { withEndoAgent } from '../context.js';
+import { parsePetNamePath } from '../pet-name.js';
 
 export const open = async ({ webletName, agentNames }) => {
   await withEndoAgent(agentNames, { os, process }, async ({ agent }) => {
-    const weblet = E(agent).lookup(webletName);
+    const webletNamePath = webletName && parsePetNamePath(webletName);
+    const weblet = E(agent).lookup(...webletNamePath);
```

Comment:
`name` is required for the `open` command:
```suggestion
    const weblet = await E(agent).lookup(...parsePetNamePath(webletName));
```